### PR TITLE
added buttons for title and subtitle, added icons for bold, italic and underline

### PR DESCRIPTION
**Pull request by i-fer**
… props. Style.js: added styled icons for bold, italic and underline. Changed from a to button on styles component

__ISSUE_REF__

### Issue

added icons an buttons for title, subtitle, added icons for bold, icon and underline

### Acceptance Criteria

user is able to see buttons on the toolbar

### QA Steps

_Give a brief list of steps to QA the changes in this PR._

### CODE REVIEW

File: src/style.js [hunk 1]
```diff
@@ -25,3 +32,21 @@ export const Toolbar = styled.div`
     margin-left: 15px;
   }
 `;
+
+export const BoldIcon = styled(Bold)`
```
Comment (qmfkdldks) [inline src/style.js:36]: Great! :+1:

File: src/editor.js [hunk 2]
```diff
@@ -14,9 +14,21 @@ const VividEditor = ({ initialValue }) => {
...
+        <MarkButton >
+          Subtitle
+        </MarkButton>
+        <MarkButton format="bold">
+          <BoldIcon/>
+        </MarkButton>
+        <MarkButton format="italic">
+          <ItalicIcon/>
+        </MarkButton>
+        <MarkButton format="code">
```
Comment (qmfkdldks) [inline src/editor.js:29]: I think format should be underline instead of the code.

**Review by qmfkdldks — CHANGES_REQUESTED** [review]
Can you run these commands and resolve all issues (tests and lint)?
And does mark buttons work as expected?

```shell
git checkout master
git pull origin master
git checkout adding-sneak-animation
git rebase master

yarn test
yarn test:lint
```

**Comment by i-fer** [comment]
> Can you run these commands and resolve all issues (tests and lint)?
> And does mark buttons work as expected?
> 
> ```shell
> git checkout master
> git pull origin master
> git checkout adding-sneak-animation
> git rebase master
> 
> yarn test
> yarn test:lint
> ```


where it says "git checkout adding-sneak-animation" should it be "git checkout add  addToolbarButtons"  ?

**Comment by qmfkdldks** [comment]
> > Can you run these commands and resolve all issues (tests and lint)?
> > And does mark buttons work as expected?
> > ```shell
> > git checkout master
> > git pull origin master
> > git checkout adding-sneak-animation
> > git rebase master
> > 
> > yarn test
> > yarn test:lint
> > ```
> 
> where it says "git checkout adding-sneak-animation" should it be "git checkout add addToolbarButtons" ?

You are right. Can you rebase from master and fix all lint issues?

> > Can you run these commands and resolve all issues (tests and lint)?
> > And does mark buttons work as expected?
> > ```shell
> > git checkout master
> > git pull origin master
> > git checkout adding-sneak-animation
> > git rebase master
> > 
> > yarn test
> > yarn test:lint
> > ```
> 
> where it says "git checkout adding-sneak-animation" should it be "git checkout add addToolbarButtons" ?

> > Can you run these commands and resolve all issues (tests and lint)?
> > And does mark buttons work as expected?
> > ```shell
> > git checkout master
> > git pull origin master
> > git checkout adding-sneak-animation
> > git rebase master
> > 
> > yarn test
> > yarn test:lint
> > ```
> 
> where it says "git checkout adding-sneak-animation" should it be "git checkout add addToolbarButtons" ?

You are right. Can you rebase from master and fix all lint issues?

**Review by qmfkdldks — APPROVED** [review]
Thank you
Have you checked if the buttons work fine? (whether it change the text with the corresponding element)
Should check `src/components/Leaf`

**Pull request merged**